features/cli/deploy.feature

```diff
@@ -756,6 +756,11 @@ Feature: deployment related features
     Then the step should succeed
     And the output should contain:
       | deployment.kubernetes.io/revision:2 |
+    When I run the :get client command with:
+      | resource      | deployment                |
+    Then the step should succeed
+    And the output should contain:
+      | READY |
```

Comment:
better use somehtinglike:
```
Then the the expression should be true> deployment("deployment-name").ready?
```

Reply:
keep beating me to the punch today. :)

Reply:
@zhouying7780 , LGTM, thanks. Like https://github.com/openshift/origin/pull/22572#discussion_r278029486 , "It is always helpful to write a PR description stating why we need this change" is always recommended.
@akostadinov this PR (and https://github.com/openshift/cucushift/pull/6973 ) are added for https://bugzilla.redhat.com/show_bug.cgi?id=1608448 (MSTR-189), so it LGTM
Sorry for delayed review due to 4.1 higher : )

Reply:
Can we just use the method explained above instead of string check? This was the suggestion.

Reply:
Above mentioned https://bugzilla.redhat.com/show_bug.cgi?id=1608448 (MSTR-189) is change of **display** of output for oc get sts/deployment. Author made the PRs to track it.

Reply:
In such case it is much better to put this test into unit tests. But if you still believe we need it here, I can merge.